### PR TITLE
DEVOPS-900 - Support integration with Bitwarden Events

### PR DESCRIPTION
**Pull request by vgrassia**
- Update app to fit Splunkbase's requirements.
- Update `build` workflow to produce proper artifacts for direct upload to Splunkbase.

### CODE REVIEW

File: app/bitwarden_event_logs/default/collections.conf [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [eventsapi]
-last_log_date = time
\ No newline at end of file
+field.last_log_date = time
```
[the remark this answers is not in the document]
Comment (tangowithfoxtrot) [reply app/bitwarden_event_logs/default/collections.conf:2]: Either way, the `last_log_date` is getting updated/created by the integration.

**Review by tangowithfoxtrot — APPROVED** [review]
Tested on an existing configuration of the Splunk integration and a fresh installation with cloud and self-hosted Bitwarden organizations. Event logs with the Linux amd64 artifacts produced by this build.

Looks great! 🚀

File: app/bitwarden_event_logs/default/collections.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (vgrassia) [reply app/bitwarden_event_logs/default/collections.conf:2]: I honestly can't remember, I believe it's from their documentation.  That specific commit was months ago, these changes were never merged in.

[the remark this answers is not in the document]
Comment (tangowithfoxtrot) [reply app/bitwarden_event_logs/default/collections.conf:2]: Gotcha. I was mostly concerned about it not updating that field in the Splunk API, but after testing, it works as expected.

**Pull request merged**